AnnoyancesFilter/Popups/sections/push-notifications_general.txt

```diff
@@ -106,7 +106,6 @@
 ||api.sociaplus.com^$third-party
 ||api.useinsider.com^$third-party
 ||app.dengage.com^$third-party
-||app.getbeamer.com^$third-party
```

Comment:
Why this was removed?

Reply:
@zloyden Because this rules breaks this [website](https://www.browserstack.com/release-notes) function.

Reply:
It's best to make an exception first.
```adblock
@@||app.getbeamer.com^$domain=browserstack.com
```